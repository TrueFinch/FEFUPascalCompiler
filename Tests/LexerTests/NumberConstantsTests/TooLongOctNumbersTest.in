&12345123451234523456712371351253