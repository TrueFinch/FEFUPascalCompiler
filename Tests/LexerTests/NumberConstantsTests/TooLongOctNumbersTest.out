Error on (1,1) in &12345123451234523456712381351253: Value was either too large or too small for a UInt64.